### PR TITLE
added balanced m9k assault rifles

### PR DESCRIPTION
**Pull request by StrawWagen**
looking for opinions on how i did this, before I start on the other packs
also changed formatting on the minigun to match new formatting

### CODE REVIEW

**Comment by wrefgtzweve** [comment]
local weapon = cfcEntityStubber.getWeapon( "weapon_class" )

File: lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak47.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+if SERVER then AddCSLuaFile() end
+
+CFC_M9k_Stubber.registerStub( function()
+    local weapon = weapons.GetStored( "m9k_ak47" )
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak47.lua:4]: on almost all files, use `local SWEP = cfcEntityStubber.GetStored( "m9k_ak47" )` instead

Comment (StrawWagen) [reply lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak47.lua:4]: pain

**Comment by wrefgtzweve** [comment]
Yeah, small thing i just realised you're using `CFC_M9k_Stubber.registerStub` for all the functions, we're not using M9k Stubber anymore thus it's `cfcEntityStubber.registerStub`

File: lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak74.lua [hunk 2]
```diff
@@ -0,0 +1,11 @@
+AddCSLuaFile() 
+
+CFC_M9k_Stubber.registerStub( function()
+    local SWEP = cfcEntityStubber.getWeapon( "m9k_ak74" )
+
+    SWEP.Spawnable = false
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak74.lua:6]: why?

File: lua/cfc_entity_stubber/m9k/assault_rifles/m9k_acr.lua [hunk 3]
```diff
@@ -0,0 +1,14 @@
+if SERVER then AddCSLuaFile() end
+
+CFC_M9k_Stubber.registerStub( function()
+    local SWEP = cfcEntityStubber.GetStored( "m9k_acr" )
+
+    SWEP.Primary.RPM			    = 825
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/assault_rifles/m9k_acr.lua:6]: Styling like this is unnecessary

File: lua/cfc_entity_stubber/m9k/assault_rifles/m9k_acr.lua [hunk 4]
```diff
@@ -0,0 +1,14 @@
+if SERVER then AddCSLuaFile() end
+
+CFC_M9k_Stubber.registerStub( function()
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/assault_rifles/m9k_acr.lua:3]: All these files are using `CFC_M9k_Stubber.registerStub` this wont work.
Use `cfcEntityStubber.registerStub` like shown in the examples.

**Review by wrefgtzweve — CHANGES_REQUESTED** [review]
Stub files won't work in their current state and some small style / overlook corrections.

File: lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak47.lua [hunk 5]
```diff
@@ -0,0 +1,15 @@
+AddCSLuaFile()
+
+CFC_M9k_Stubber.registerStub( function()
+    local SWEP = cfcEntityStubber.getWeapon( "m9k_ak47" )
+
+    SWEP.Primary.RPM                = 600
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/assault_rifles/m9k_ak47.lua:6]: Quick note, styling like this increases file size by ~20-30% is it worth it?

**Pull request closed without merging**